test/runtests.jl

```diff
@@ -1,6 +1,6 @@
 using Nemo
 using Test
-using InteractiveUtils
+using Nemo.InteractiveUtils
```

Comment:
Huh, what is `InteractiveUtils` being used for, anyway? In the tests, and also in Nemo itself?

Reply:
ok it seems the tests use `@which` in a few place. But why does Nemo itself do `using InteractiveUtils` ? @thofma do you remember (you introduced it in e8b6dc52b960597bc1d1c9d6bb9dfca9868eef4f, almost five years ago, so I'd not be surprised if you don't ;-) )

Reply:
Hhm, good question. Local test seems to succeed without it. I'll push it in a minute.

Reply:
Let's see if CI succeeds without it (c3b629db0779a46395134825a2de968a5894440c). In case thofma cannot remember, we will have some certainty nothing breaks.

Reply:
I only remember the usage in the tests. Probably fine to remove it.